### DISCUSSION TITLE
I'm very upset about images used in this example

### DISCUSSION THREAD

**Discussion by Toshinaki** [mantinedev/mantine#2170] [Feedback]
https://mantine.dev/others/carousel/#images-carousel

Where is Charizard or Pikachu or, "Mantine"?
![image](https://user-images.githubusercontent.com/10347466/185631919-e8e41da2-fcad-4c95-b81b-0ce733ad6507.png)

**Comment by rtivital** [comment]
Better not get greedy, we already have the name, nintendo can punish you for using their ip

**Reply by Toshinaki** [reply]
💔
![image](https://user-images.githubusercontent.com/10347466/185633468-6ffd94bb-1255-42e9-b549-515232728417.png)

**Reply by Toshinaki** [reply]
Just joke :P
https://mantine.dev/others/rte/#images-and-videos-embeds
I saw this example and kind of worried.
